src/test/scala/nl.knaw.dans.easy.deposit/docs/DDMSpec.scala

```diff
@@ -447,7 +443,7 @@ class DDMSpec extends TestSupportFixture with DdmBehavior {
         <ddm:accessRights>OPEN_ACCESS</ddm:accessRights>
       </ddm:profile>
       <ddm:dcmiMetadata>
-        <dcterms:identifier xsi:type="id-type:DOI">doi:10.17632/DANS.6wg5xccnjd.1</dcterms:identifier>
+        <dcterms:identifier xsi:type="id-type:DOI">10.17632/DANS.6wg5xccnjd.1</dcterms:identifier>
```

Comment:
@rvanheest 
I don't get the prefix from `curl -i -X POST 'http://deasy.dans.knaw.nl:20140/create?type=doi'`
and there is also no prefix when downloading an XML from (for example) https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:112326/tab/1
The api.yml is not specific on this point.